debian/control

```diff
@@ -1,21 +1,30 @@
-Source: pulseaudio-modules-droid-hidl-28
+Source: pulseaudio-modules-droid-hidl
 Section: sound
 Priority: optional
 Maintainer: Danct12 <danct12@disroot.org>
 Build-Depends: debhelper (>= 7),
                dh-autoreconf,
                libdbus-1-dev,
+               android-headers-24,
                android-headers-28,
-               pulseaudio-modules-droid-dev,
+               pulseaudio-modules-droid-24-dev,
+               pulseaudio-modules-droid-28-dev,
                pulseaudio-dev,
                libpulse-dev,
                libltdl-dev,
                audiosystem-passthrough-dev
 Standards-Version: 3.8.4
 
+Package: pulseaudio-modules-droid-hidl-24
```

Comment:
You do not need to build it with android-headers-24. This module relies on binderized HIDL interfaces which became available in Android 8, and while it may compile, it will not work on earlier devices.

Reply:
Ok, will remove building for -24. I thought audiosystem-passthrough also work as AudioFlinger on Android 7 too (given that AudioFlingerGlue is stopped).